.circleci/config.yml

```diff
@@ -195,6 +195,7 @@ jobs:
           condition:
             equal: [*nodejs_current, << parameters.nodejs >>]
           steps:
+            - run: npm run prettier-ci
```

Comment:
This works as shown here 🤣 https://app.circleci.com/pipelines/github/mozilla/web-ext/1653/workflows/f4c65822-c0c2-4d21-8a50-9a7600c3648c/jobs/6694